src/parachain/issue.ts

```diff
@@ -34,13 +36,33 @@ export function encodeIssueRequest(req: IssueRequest, network: Network): IssueRe
  */
 export interface IssueAPI {
     /**
-     * Send an issue request transaction
-     * @param amountSat PolkaBTC amount (denoted in Satoshi) to issue
-     * @param vaultId (optional) Request the issue from a specific vault. If this parameter is unspecified,
-     * a random vault will be selected
-     * @returns An object of type {issueId, vault} if the request succeeded. The function throws an error otherwise.
+     * Gets the threshold for issuing with a single vault, and the maximum total
+     * issue request size. Additionally passes the list of vaults for caching.
+     * @returns An object of type {singleVault, maxTotal, vaultsCache}
      */
-    request(amountSat: PolkaBTC, vaultId?: AccountId, griefingCollateral?: DOT): Promise<IssueRequestResult>;
+    getRequestLimits(): Promise<IssueLimits>;
+
+    /**
+     * Request issuing of PolkaBTC.
+     * @param amountSat PolkaBTC amount (denoted in Satoshi) to issue.
+     * @param availableVaults (optional) A list of all vaults usable for issue, to avoid re-querying the parachain
+     * @returns An array of type {issueId, vault, error} if the requests succeeded. error will be null for successful
+     * requests; issueId and vault will be null for failed ones.
+     */
+    request(amountSat: PolkaBTC, availableVaults?: Map<AccountId, PolkaBTC>): Promise<IssueRequestResult[]>;
+
+    /**
+     * Send a batch of aggregated issue transactions (to one or more vaults)
+     * @param amountSat PolkaBTC amounts (denoted in Satoshi) to issue using each vault
+     * @param vaultId The vaults from which to request issue
+     * @returns An array of type {issueId, vault, error} if the requests succeeded. error will be null for successful
+     * requests; issueId and vault will be null for failed ones.
+     */
+    requestAdvanced(
```

Comment:
maybe a better name would be `batchRequest`?

Reply:
My reasoning behind this is that this is basically what would be called if someone wants to make sure to manually pass every parameter. It doesn't have to be used for batching, either - if you want to request with just a single vault, but you want to specify exactly which vault to use, you'd call `requestAdvanced` rather than `request`. E.g. for the power-user mode you mentioned.

Conversely, you can also batch request by calling `request` (it just auto-selects the vaults for you, if necessary).

Reply:
Hmm, I'm not thrilled with this naming since it doesn't better describe the difference between this and the other `request` method.

Reply:
Would you have a better suggestion? In my mind the main difference is that `request` auto-selects everything with just an amount, while `requestAdvanced` requires specifying what you want to request and where and doesn't set any parameters itself.

In the past there was also griefing rate, now that it's removed maybe `requestWithVaultList` is a bit more accurate, but I don't like that name. Perhaps something like `requestManually`.